### ISSUE TITLE
Feature request: Network support, e.g. Samba drives

### ISSUE THREAD

**Issue by StefanPofahl** [alphaonex86/Ultracopier#90]
After some hurdles and with your help, I manged to get it into operation :-)

I can see the Floppy-Disk-Icon in the status line :-)
But copy files to my NAS does not seem to be possible :-(

How can I access a folder with the network address similar to that?
```
smb://NAS/Multimedia/video
```
Another feature request would be, to make USB-drives more easy accessible.
Unfortunately, I cannot see an icon of an attached USB-device in the `ultracopier` file-browser.

Positive:
Copy of a larger file on to the USB device worked fast and successful! :-)

**Comment by alphaonex86** [comment]
Hi, no KIO is not supported util dolphin integrate the support of ultracopier, only local file

**Comment by StefanPofahl** [comment]
Does this mean in other Linux distributions I can see USB-Drives and Network-Locations?

And what about the file explorer `Files/Nemo/Nautilus`? Can I link `ultracopier` to other file-browsers that support `ultracopier`?

**Comment by alphaonex86** [comment]
Hi, right, I code local file only support (via posix API, open()/raed()/write()/close())
No one file browser now support Ultracopier on Linux

**Comment by StefanPofahl** [comment]
Belief me or not, but with my current build of your software I can see the Icons of USB devices and network folders pre-defined as bookmarks. I can select a network folder as destination and the copy process starts, but once the copy process is accomplished I cannot see any files in the destination folder.
If I repeat the same copy-procedure, `ultracopier` asks me what to do, if the files in the destination should be replaced,
but unfortunately only `ultracopier` "sees" / detect the files in the destination folder.

Visible bookmark of network folders:
![ultracopier_network_visible](https://github.com/alphaonex86/Ultracopier/assets/65159315/8214f77c-ae15-4e84-b614-22a93f1aae28)

Copy-task to network folder in progress:

![ultracopier_copy_to_network_folder](https://github.com/alphaonex86/Ultracopier/assets/65159315/7b6c22a9-05d9-4955-acf2-03dc12c71ad5)

**Comment by alphaonex86** [comment]
yes, treat smb: as local folder, not as remote folder, mean into the current folder you will have now and smb: folder into the computer where you use ultracopier

**Comment by alphaonex86** [comment]
I don't have better behavior, this can be the behavior excepted by the user.

If I can have simple bug case, this will help me to reproduce and fix, maybe this bug is only into your computer.

**Comment by StefanPofahl** [comment]
If the destination folder contains a string like `smb://` a copy into a different folder into the current program folder should be avoided. The GUI indicates that the destination folder is `smb://[...]`, but instead it is a relative path: `./smb/[...]`

**Comment by alphaonex86** [comment]
Hi, for now protocol not supported. I don't know how do it with KIO
